server/adaptors/integrations/types.ts

```diff
@@ -50,38 +52,19 @@ interface SerializedIntegrationStatics {
   darkModeGallery?: SerializedStaticAsset[];
 }
 
-interface IntegrationAssets {
-  savedObjects?: {
-    name: string;
-    version: string;
-  };
-  queries?: Array<{
-    name: string;
-    version: string;
-    language: string;
-  }>;
+interface IntegrationAsset {
+  name: string;
+  version: string;
+  extension: string;
+  type: SupportedAssetType;
```

Comment:
This is the core change, most of the refactoring is following the consequences of this type update